etc/db_schema_department_table.sql

```diff
@@ -0,0 +1,9 @@
+/* Department table */
+CREATE TABLE Department (
+id int AUTO_INCREMENT,
```

Comment:
nit: maybe tab indent each field to show that it is part of the same statement, same as Class table.